index.js

```diff
@@ -141,9 +141,9 @@ module.exports = function (file, options) {
       for (var i = node.range[0] + 1; i < node.range[1]; i++) {
           chunks[i] = '';
       }
-      paths.forEach(function (p) {
+      paths.forEach(function (p, i) {
         var st = '\nrequire(' + p + ')'
-        chunks[node.range[1] + 1] = st
+        chunks[node.range[1] + (i + 1)] = st
```

Comment:
i think if it hits the end of the empty chunks it'll start overwriting other code.